File: gateway/src/main/java/com/beat/gateway/guest/internal/config/GuestAccessConfig.java
package com.beat.gateway.guest.internal.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

import com.beat.gateway.guest.internal.GuestAccessThrottleService;
import com.beat.gateway.guest.internal.GuestPasswordHashService;
import com.beat.gateway.guest.internal.GuestSessionService;
import com.beat.gateway.shared.internal.config.RedisConfig;

@Configuration(proxyBeanMethods = false)
@Import({
	RedisConfig.class,
	GuestSessionService.class,
	GuestPasswordHashService.class,
	GuestAccessThrottleService.class
})
public class GuestAccessConfig {
}

File: gateway/src/main/kotlin/com/beat/gateway/guest/internal/config/GuestAccessConfig.kt
package com.beat.gateway.guest.internal.config

import com.beat.support.security.password.internal.BCryptPasswordHasher
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import

@Configuration(proxyBeanMethods = false)
@Import(BCryptPasswordHasher::class)
class GuestAccessConfig
